akka-docs/src/main/paradox/typed/guide/diagrams/actor_top_tree.svg

```diff
@@ -1,365 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
isn't this the source of actor_top_tree.png?

Reply:
Not any more, i redrew it in keynote, couldn't get any reasonable svg-editor to work.